austation/code/modules/food_and_drinks/food/tablecraft/recipes_bread.dm

```diff
@@ -0,0 +1,23 @@
+/datum/crafting_recipe/food/chef_only/supermatterbread
+	name = "Supermatter bread (Chef only)"
+	reqs = list(
+		/obj/item/reagent_containers/food/snacks/store/bread/plain = 5, // the supermatter aint small
+		/obj/machinery/power/supermatter_crystal/engine = 0, // supermatter not an item, it is handled in the check_requirement proc
+	)
+	result = /obj/item/reagent_containers/food/snacks/store/bread/supermatter
+	subcategory = CAT_BREAD
+
+/datum/crafting_recipe/food/chef_only/supermatterbread/check_requirements(mob/user, list/collected_requirements) //checks to see if the crafter is a chef
+	var/mob/living/carbon/human/H = user
```

Comment:
Should probably have a type check on this, even if I'm only aware of humans being able to use crafting

Reply:
yep, only humans can craft, there's no need for this check